src/Config/ConfigLocator.php

```diff
@@ -160,15 +158,15 @@ public function sources()
      *
      * @return string[]
      */
-    public function configFilePaths()
+    public function configFilePaths(): array
```

Comment:
Should we change these array hints to Iterable per https://steemit.com/php/@crell/php-never-type-hint-on-arrays?

Reply:
Can we do that as a follow up? My intent is to keep the non-scripted changes low, and to do multiple passes with multiple PRs.

Reply:
Yes, makes sense.